swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
+from swap_meet.item import Item
+
 class Clothing:
-    pass
\ No newline at end of file
+    def __init__(self, condition = 0):
+        self.category = "Clothing"
+        self.condition = condition
+        self.condition_description = 4
```

Comment:
Clothing should be a subclass of Item.  If Clothing is a subclass of Item, it would have access to the `condition_description` function in `Item`.  It could also use the parent class `__init__` function to simplify the code here:
```suggestion
    def __init__(self, condition = 0):
        super().__init__("Clothing", condition)
```